app/services/idv/steps/doc_auth_base_step.rb

```diff
@@ -31,7 +31,7 @@ def idv_failure(result)
           redirect_to idv_session_errors_exception_url
         else
           @flow.analytics.track_event(
-            Analytics::IDV_DOC_AUTH_WARNING_VISITED,
+            'IdV: doc auth warning visited',
             step_name: self.class.name,
             remaining_attempts: throttle.remaining_count,
           )
```

Comment:
this should probably be `@flow.analytics.idv_doc_auth_warning_visisted(`

Reply:
Nice catch, this has been updated.